src/python_testing/TC_WEBRTCP_2_2.py

```diff
@@ -108,7 +108,7 @@ async def test_TC_WebRTCProvider_2_2(self):
         resp = await self.send_single_cmd(cmd=cmd, endpoint=endpoint, payloadCapability=ChipDeviceCtrl.TransportPayloadCapability.LARGE_PAYLOAD)
         asserts.assert_equal(type(resp), Clusters.WebRTCTransportProvider.Commands.SolicitOfferResponse,
                              "Incorrect response type")
-        asserts.assert_not_equal(resp.webRTCSessionID, 0, "webrtcSessionID in SolicitOfferResponse should not be 0.")
+        asserts.assert_equal(resp.webRTCSessionID, 0, "webrtcSessionID in SolicitOfferResponse should be 0.")
```

Comment:
same here: do test plans require session 0 based on how we use things? If any session is valid, what is the reason we expect exactlu 0 here and in other places? That does seem to prevent re-running tests without factory reset. A comment should explain or we should drop these checks.